### ISSUE TITLE
Mapping between lists of types with dictionary property throws InvalidOperationException

### ISSUE THREAD

**Issue by anton-g** [agileobjects/AgileMapper#24]
I think I may have run into a bug:

```c#
    public class Test1
    {
        public Dictionary<string, string> Options { get; set; }
    }

    public class Test2
    {
        public Dictionary<string, string> Options { get; set; }
    }

    var plan = Mapper.GetPlanFor<List<Test1>>().ToANew<List<Test2>>();
```
The above results in: `System.InvalidOperationException: 'No coercion operator is defined between types 'Test1' and 'System.Collections.Generic.List'1[Test1]'.'`

Also; thank you for working on AgileMapper!

**Comment by SteveWilkes** [comment]
This is fixed in the latest code - release to follow. Thanks for the feedback! :)

**Comment by SteveWilkes** [comment]
This is fixed in [release 0.13](https://www.nuget.org/packages/AgileObjects.AgileMapper/0.13.0) - apologies for the delay, I was out of the country :)
